### ISSUE TITLE
Collapsible List Accessibility - Keyboard Interaction Bug

### ISSUE THREAD

**Issue by HannahRedmond-Velo** [jamesmfriedman/rmwc#521]
RMWC 5.7.0

When traversing a RMWC `collapsible list` using keyboard navigation alone, the list items have a tendency to get 'stuck' meaning that the user is unable to navigate past a collapsible list handle and allow focus to rest on the next item in the list. 

**Replication**

The RMWC (https://rmwc.io/) site navigation uses the collapsible list component and can be used to demonstrate the issue:

- Highlight the ‘getting started’ item in the site navigation
- Press DOWN arrow to move the focus to the ‘Avatars’ item
- Press Up arrow to move the focus back to the ‘getting started’ item
- Pressing the DOWN arrow again does nothing and the user can no longer traverse the collapsible list using the keyboard alone. 

The issue can be cleared by expanding the ‘getting started’ item and selecting one of the sub menu items

There are various ways to trigger the issue, additionally - 

- Highlight the ‘getting started’ item
- Click enter to expand the ‘getting started’ menu
- Click enter again to collapse the menu
- Press DOWN arrow - nothing happens. The user is stuck until they re-open the sub menu item and select one of the items. 

NB: If the Collapsible list parent is further down the menu, the user is able to use the arrow UP, and down, but not beyond the menu item which was last interacted with

**Comment by jamesmfriedman** [comment]
Thanks for the simple repro steps. I’ll look into this, probably a simple fix.

**Comment by robinjac** [comment]
Hi, i encountered this issue recently. Has it been resolved? Otherwise I'll gladly look into it.
